docs/content/preview/api/ysql/syntax_resources/the-sql-language/statements/drop_index.grammar.md

```diff
@@ -0,0 +1,4 @@
+```output.ebnf
+drop_index ::= DROP INDEX [ IF EXISTS ] index_name [ ... ] 
```

Comment:
drop multiple objects is not supported.  Also, the syntax is wrong because of missing comma.

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/17543/commits/33584556a4beff4ad6b2eca572361c6381deda35